test/Peaky.Client.Tests/PeakyClientTests.cs

```diff
@@ -158,16 +158,16 @@ public class PeakyClientTests
         }
 
         [Fact]
-        public async void It_retrieves_no_test_from_Url()
+        public async void When_the_response_contains_no_peaky_tests_Then_it_returns_an_empty_list()
         {
             using (var server = new FakeHttpService.FakeHttpService()
                 .WithTestResultAt("/no_test", "hijklmnop", false))
             {
                 var client = new PeakyClient(new Uri(server.BaseAddress, @"/tests"));
 
-                var test = new Test("", "", new Uri(server.BaseAddress, "/no_test"), null);
+                var tests = client.GetTests();
 
-                test.Should().Equals(Array.Empty<Test>());
+                tests.Should().Equals(Array.Empty<Test>());
```

Comment:
This test is still evergreen.

`tests.Should().Equals(Array.Empty<Test>());` is equivalent to:
```
    var left = tests.Should(); // this is just an object of type  ObjectAssertions
    var right = Array.Empty<Test>();
    left.Equals(right); // this is an object equality comparison, not an assertion. It simply returns false, but nothing reacts to it.
```

try `tests.Should().BeEmpty()` instead.